src/v/kafka/server/snc_quota_manager.cc

```diff
@@ -19,6 +19,8 @@ using namespace std::chrono_literals;
 
```

Comment:
> k/quotas: throughput limits bindings and quotas as inoutpairs
Use inoutpair to represent TP limit property binding, and the quota
token buckets.

commit message needs updated for type name change